python/src/ExpectationSimulationAlgorithm_doc.i.in

```diff
@@ -11,70 +11,262 @@ X : :class:`~openturns.RandomVector`
+size which satisfies pre-defined stopping rules. 
+It increases the sample size :math:`n` incrementally until a stopping criterion is met. 
+
+The algorithm is based on two nested loops:
+
+- the outer loop sets the number of iterations of the algorithm; 
+  this can be configured using :meth:`setMaximumOuterSampling`,
+
+- the inner loop sets the number of function calls which can be parallelized; 
+  this can be configured using :meth:`setBlockSize`.
+
+The algorithm allows to get the best possible performance on distributed 
+supercomputers and multi-core workstations. 
+For example, if the block size is equal to 100, then the sample size 
+is successively equal to 100, 200, etc... 
+Each block of evaluation of the outputs can be parallelized, which may improve the performance of the algorithm.
+We suggest to set the block size as a multiple of the number of cores. 
+
+The algorithm stops either when the maximum number of outer iterations is reached or when the target precision is met. 
+The latter stopping criterion is based on either the coefficient of variation of the 
+sample mean (relative criterion) or the standard deviation of the sample 
+mean (absolute criterion). 
+
+Let :math:`\vect{X}\in\Rset^{n_X}` be the random vector. 
+We consider :math:`\vect{n}` independent realizations :math:`\vect{x}^{(1)}`, 
+..., :math:`\vect{x}^{(n)}\in\Rset^{n_X}` of the random vector :math:`\vect{X}`. 
+We estimate :math:`\Expect{\vect{X}}` with the sample mean:
+
+.. math::
+
+    \overline{\vect{x}} = \frac{1}{n} \sum_{i=1}^n \vect{x}^{(j)}.
+
+We estimate :math:`\Var{X_i}` for :math:`i = 1, \ldots, n_X` 
+with the unbiased sample variance:
+
+.. math::
+
+    \hat{\sigma}^2_i = \frac{1}{n - 1} \sum_{j=1}^n \left(x^{(j)}_i - \overline{x}_i\right)^2.
+
+For :math:`i = 1`, the expected value of the sample mean is:
+
+.. math::
+
+    \Expect{\overline{X}_i} = \Expect{X_i}
+
+and, since the observations are independent, the variance of the sample mean is:
+
+.. math::
+
+    \Var{\overline{X}_i} = \frac{\Var{X_i}}{n}.
+
+Moreover, we can estimate the standard deviation of the sample mean with:
+
+.. math::
+
+    \hat{\sigma}\left(\overline{X}_i\right) = \frac{\hat{\sigma}_i}{\sqrt{n}}
+
+for :math:`i = 1, \ldots, n_X`.
+
+If the expectation :math:`\Expect{X_i}` is nonzero, the coefficient of variation of the sample mean is:
+
+.. math::
+
+    CV\left(\overline{X}_i\right) = \frac{\sqrt{\Var{\overline{X}_i}}}{\left|\Expect{\overline{X}_i}\right|} = \frac{\sqrt{\Var{X_i}}/\sqrt{n}}{\left|\Expect{X_i}\right|}
+
+for :math:`i = 1, \ldots, n_X`.
+
+We can estimate it with
+
+.. math::
+
+    \widehat{CV}\left(\overline{X}_i\right) = \frac{\hat{\sigma}_i/\sqrt{n}}{\left|\overline{x}_i\right|}
+
+for :math:`i = 1, \ldots, n_X`.
+
+When the sample size :math:`n` increases, the sample standard deviation and the sample coefficient of variation decrease to zero at the Monte-Carlo rate of :math:`1/\sqrt{n}`. 
 
```

Comment:
It is not true: the convergence is monotone in a very relaxed way. For a given simulation, you see on any convergence graph that the standard deviation behavior is quite chaotic. It is the expected value of the standard deviation of the estimate which is decreasing. The same is true for the coefficient of variation.

Reply:
You are right. Fixed: "When the sample size n increases, the expected value of the sample standard deviation and the expected value of the sample coefficient of variation decrease to zero. "